main/zsh/APKBUILD

```diff
@@ -8,10 +8,10 @@
 #     - CVE-2018-1071
 #
 pkgname=zsh
-pkgver=5.6.2
+pkgver=5.7.1
 pkgrel=0
 pkgdesc="Very advanced and programmable command interpreter (shell)"
-url="http://www.zsh.org"
+url="https://www.zsh.org"
 arch="all"
 license="BSD"
```

Comment:
license= is not an SPDX identifier and is ambiguous

Reply:
Rebased